website/achievements.py

```diff
@@ -187,12 +188,12 @@ def check_code_achievements(self, code, level):
         if 'hedy_honor' not in self.achieved and "Hedy" in code:
             self.new_achieved.append("hedy_honor")
         if 'hedy-ious' not in self.achieved:
-            lines = code.splitlines()
-            for line in lines:
-                if "print" in hedy.all_commands(line, level, self.lang):
-                    if lines.count(line) >= 10:
-                        self.new_achieved.append("hedy-ious")
-                        return
+            all_print_arguments = hedy.all_print_arguments(code, level, self.lang)
+            for argument in all_print_arguments:
+                if all_print_arguments.count(argument) > 10:
```

Comment:
I think this should be `>=` instead of `>`, otherwise it all seems to work fine!

Reply:
ah good catch, changed!